Framework/PythonInterface/mantid/api/src/Exports/FunctionFactory.cpp

```diff
@@ -24,6 +24,39 @@ using namespace boost::python;
 
 GET_POINTER_SPECIALIZATION(FunctionFactoryImpl)
 
+namespace Mantid {
+namespace PythonInterface {
+
+/// Specialization for IFunction. It needs to pass a bool to the created
```

Comment:
This is good - I like that we can hide this from users. Could you update the comment to explain a little bit about how the mechanism works?